low-level/sensors/impl/generic/node_hw.py

```diff
@@ -573,12 +574,18 @@ def _run_ipmitool_subcommand(self, subcommand, grep_args=None, out_file=subproce
                     resstr = b''.join([val for val in res if val])
                     resstr = resstr.decode(self.IPMI_ENCODING)
                     if resstr.find(self.IPMI_ERRSTR) == 0:
-                        logger.error(f"{self.SENSOR_NAME}: {ipmi_tool} error:: {resstr}\n \
-                            Dependencies failed, shutting down sensor")
+                        logger.error(f"{self.SENSOR_NAME}: {ipmi_tool} \
+                            error:: {resstr}\n Dependencies failed, \
+                            shutting down sensor")
                         self.request_shutdown = True
+                self.iem.iem_fault("IPMITOOL_ERROR")
             elif (retcode == BASH_ILLEGAL_CMD):
-                logger.error(f"{self.SENSOR_NAME}: Required ipmitool missing on Node. Dependencies failed, shutting down sensor")
+                logger.error(f"{self.SENSOR_NAME}: Required ipmitool missing \
+                    on Node. Dependencies failed, shutting down sensor")
                 self.request_shutdown = True
+                self.iem.iem_fault("IPMITOOL_ERROR")
+        else:
+            self.iem.iem_fault_resolved("IPMITOOL_ERROR", "IPMITOOL_AVAILABLE")
```

Comment:
same comment as udisks, track with an in-memory flag to know whether to raise fault-resolved alert or not

Reply:
Done.